src/python/grpcio_csds/grpc_csds/__init__.py

```diff
@@ -31,14 +34,14 @@ def FetchClientStatus(request, unused_context):
         )
 
     @staticmethod
-    def StreamClientStatus(request_iterator, context):
+    def StreamClientStatus(request_iterator: Iterable[Any], context: Any):
```

Comment:
No return type?

Reply:
In general, `Any` is a sign of a type that is too broad. Should this be `Iterable[ClientStatusRequest]`? Same for context. Surely there's a better type to use.